openstudiocore/src/energyplus/Test/ScheduleInterval_GTest.cpp

```diff
@@ -144,9 +147,112 @@ TEST_F(EnergyPlusFixture,ForwardTranslator_ScheduleFixedInterval_Hourly)
   EXPECT_EQ(8760, numUntils);
 }
 
+TEST_F(EnergyPlusFixture, ForwardTranslator_ScheduleFixedInterval_Hourly_Shifted)
+{
+  Vector values = linspace(1, 8760, 8760);
+
+  TimeSeries timeseries(DateTime(Date(MonthOfYear::Jan, 1)), Time(0, 1, 0), values, "");
+
+  Model model;
+
+  boost::optional<ScheduleInterval> scheduleInterval = ScheduleInterval::fromTimeSeries(timeseries, model);
+  ASSERT_TRUE(scheduleInterval);
+  EXPECT_TRUE(scheduleInterval->optionalCast<ScheduleFixedInterval>());
+
+  TimeSeries ts = scheduleInterval->timeSeries();
+  EXPECT_EQ(DateTime(Date(MonthOfYear::Jan, 1)), ts.firstReportDateTime());

```

Comment:
@macumber I don't think we should remove that ctor, since the problem is really the schedule. Your other suggestions are good, though. I'll add in some comments and make the test more explicit.